invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -146,6 +149,16 @@ class DatesSchema(Schema):
     description = StrippedHTML(attribute='description')
 
 
+class RightsSchema(Schema):
```

Comment:
It can inherit `VocabularyL10Schema` to avoid redefining the `id` and `title` fields.
```suggestion
class RightsSchema(VocabularyL10Schema):
```